.gitignore

```diff
@@ -13,3 +13,17 @@ DYON_tests/Read_mat.py
 *.bcf
 *Data
 examples/ECH/TCV/debug/*
+src/.metadata/*
+tcvstream
+tcvstream/
+tcvstream/*
```

Comment:
folder/

is sufficient to ignore a folder, we should have only one line to ignore files inside a folder

Reply:
Duly noted, I changed it